modules/tensor_mechanics/test/tests/umat/predef/tests

```diff
@@ -0,0 +1,90 @@
+[Tests]
+  issues = '#14974'
+  design = 'AbaqusUMATStress.md'
+  [predef]
+    type = 'Exodiff'
+    input = 'predef.i'
+    cli_args = "Materials/active='umat'"
+    exodiff = 'predef_out.e'
+    library_mode = 'DYNAMIC'
+    valgrind = 'NONE'
+    requirement = 'The Abaqus UMAT interface shall produce the same results (stress, strain, '
+                  'displacements) as the built-in MOOSE capabilities for a mechanical problem with '
+                  'an external field (not temperature) affecting material behavior '
+  []
+  [predef_reference]
+    type = 'Exodiff'
+    input = 'predef.i'
+    cli_args = "Materials/active='elasticity_tensor strain_dependent_elasticity_tensor stress "
+               "prefactor_material_block'"
+    exodiff = 'predef_out.e'
+    requirement = 'MOOSE shall avoid regression in a simple mechanical problem where a strain field '
+                  'modifies the stiffness of the material through CompositeElasticityTensor. This '
+                  'test also serves as a reference for UMAT external field verification'
```

Comment:
I don't know if those statements about "This test also serves as a reference..." really fit what's supposed to be in a reaquirement.

Reply:
I'll leave it in there. I don't think it hurts and it is valuable information